File: MiniDOMWebLibs/MiniDOM/IPInfo/CIPInfoCursor.vb
Imports minidom
Imports minidom.Databases
Imports minidom.Sistema
 
Partial Class WebSite

  
    Public Class CIPInfoCursor
        Inherits DBObjectCursorBase(Of CIPInfo)

        Private m_IP As New CCursorFieldObj(Of String)("IP")
        Private m_Descrizione As New CCursorFieldObj(Of String)("Descrizione")

        Public Sub New()
        End Sub

        Public ReadOnly Property IP As CCursorFieldObj(Of String)
            Get
                Return Me.m_IP
            End Get
        End Property

        Public ReadOnly Property Descrizione As CCursorFieldObj(Of String)
            Get
                Return Me.m_Descrizione
            End Get
        End Property

        Protected Overrides Function GetConnection() As CDBConnection
            Return LOGConn
        End Function

        Protected Overrides Function GetModule() As CModule
            Return IPInfo.Module
        End Function

        Public Overrides Function GetTableName() As String
            Return "tbl_IPInfo"
        End Function
    End Class


End Class

File: MiniDOMWebLibs1/MiniDOM/IPInfo/CIPInfoCursor.cs

namespace minidom
{
    public partial class WebSite
    {
        public class CIPInfoCursor : Databases.DBObjectCursorBase<CIPInfo>
        {
            private Databases.CCursorFieldObj<string> m_IP = new Databases.CCursorFieldObj<string>("IP");
            private Databases.CCursorFieldObj<string> m_Descrizione = new Databases.CCursorFieldObj<string>("Descrizione");

            public CIPInfoCursor()
            {
            }

            public Databases.CCursorFieldObj<string> IP
            {
                get
                {
                    return m_IP;
                }
            }

            public Databases.CCursorFieldObj<string> Descrizione
            {
                get
                {
                    return m_Descrizione;
                }
            }

            protected override Databases.CDBConnection GetConnection()
            {
                return Databases.LOGConn;
            }

            protected override Sistema.CModule GetModule()
            {
                return IPInfo.Module;
            }

            public override string GetTableName()
            {
                return "tbl_IPInfo";
            }
        }
    }
}